lib/panko/serializer_resolver.rb

```diff
@@ -1,25 +1,35 @@
 # frozen_string_literal: true
 
 class Panko::SerializerResolver
-  def self.resolve(name)
-    serializer_name = "#{name.singularize.camelize}Serializer"
-    serializer_const = safe_const_get(serializer_name)
+  class << self
```

Comment:
I took the liverty to convert to `class << self`, as it makes it easier to have private class method. Previously `is_serializer` and `safe_const_get` were actually public.